### ISSUE TITLE
glob import doesnt work togather with ~ alias

### ISSUE THREAD

**Issue by kubenstein** [parcel-bundler/parcel#3889]
First of all thank you for parcel! We love it and use it in many of our products!

# 🐛 bug report
Recently I noticed that glob importing doesnt work when the import path uses default `~` alias. currently to use glob imports we have to still use relative `../../` paths.
refs: https://parceljs.org/module_resolution.html

## 💻 Code Sample
```js
// glob importing demo
import singleImageRelative from './src/assets/images/demo.png' // works
import singleImageAlias from '~/src/assets/images/demo.png' // works
import wildcardImagesRelative from './src/assets/images/*.png' // works
import wildcardImagesAlias from '~/src/assets/images/*.png' // EMPTY object!


console.log({
  singleImageRelative,    // "/demo.cccb2033.png"
  singleImageAlias,       // "/demo.cccb2033.png"
  wildcardImagesRelative, // { demo: "/demo.cccb2033.png" }
  wildcardImagesAlias     // {} <--- this should not be an empty object
})
```
| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel           | 1.12.4
| Node             | v8.16.0
| yarn         | v1.16.0
| Operating System | macOS Mojave

**Comment by skipjack** [comment]
I _think_ I'm seeing the same bug. In short, the following doesn't work:

``` js
import flags from 'free-flag-icons/flag_background/*.jpg'
```

but a relative path does:

``` js
import flags from '../../node_modules/free-flag-icons/flag_background/*.jpg'
```

**Comment by skipjack** [comment]
Just in case anyone else stumbles on this lonely and confused 😁 

It was resolved by #4683 and the following is an example of how to re-enable it...

https://github.com/parcel-bundler/parcel/blob/bdb43afa4aaab364f4018634bc8c0d6e7fc6c30c/packages/core/integration-tests/test/integration/glob-async/.parcelrc
